__tests__/app.test.js

```diff
@@ -40,4 +40,34 @@ describe('GET /api', () => {
         expect(endpoints).toEqual(endpointsData);
       });
   });
-});
\ No newline at end of file
+});
+
+describe('GET /api/articles/:article_id', () => {
+  test('200 - responds with an article object which have selected properties', () => {
+    return request(app)
+    .get('/api/articles/1')
+    .expect(200)
+    .then(({ body }) => {
+      const { data } = body;
+      expect(typeof data).toBe('object')
+      expect(typeof data.author).toBe('string');
+      expect(typeof data.title).toBe('string');
+      expect(typeof data.article_id).toBe('number');
+      expect(typeof data.body).toBe('string');
+      expect(typeof data.topic).toBe('string');
+      expect(typeof data.created_at).toBe('string');
+      expect(typeof data.votes).toBe('number');
+      expect(typeof data.article_img_url).toBe('string');
+    })
```

Comment:
In this test, we know the exact article we are getting back, so it would be possible to assert against the specific values we expected to be returned rather than expecting type of to be string or number